source/float/floatItem.css

```diff
@@ -0,0 +1,14 @@
+@keyframes floatItem {
+  from {
+      transform: translate3d(0, 0, 0);
+  }
+  to {
+      transform: translate3d(0, -40px, 0);
+  }
+}
+
+.floatItem {
+  animation: floatItem 3s;
+  animation-direction: alternate;
+  animation-iteration-count: infinite;
```

Comment:
Being only an infinite animation is not the best way to do this. Bake the return into the keyframe itself and don't rely on the `animation-iteration-count` property as many people use the keyframes directly.

Reply:
it's not necessary  but if u will use short hand property of animation  it will be bater

Reply:
for every one